dll/win32/shell32/shlfileop.cpp

```diff
@@ -1940,7 +1940,11 @@ validate_operation(LPSHFILEOPSTRUCTW lpFileOp, FILE_LIST *flFrom, FILE_LIST *flT
             {
                 size_t cchFrom = PathAddBackslashW(szFrom) - szFrom;
                 size_t cchTo = PathAddBackslashW(szTo) - szTo;
+#ifdef __REACTOS__
```

Comment:
This `validate_operation` function is a ReactOS-specific function already (see above, already surrounded by the guards), that you have introduced in this commit https://github.com/reactos/reactos/commit/0e2bb5b0b80edac9e1078388c9d0b60fba4a55e1 .
So if what you're doing there is a bugfix, you can directly fix it without adding more REACTOS guards.

Reply:
Removed that guard.